advection_test/tracerAdvection.ipynb

```diff
@@ -0,0 +1,259 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.        if (rank == 0) and (step+1)%100 == 0 and print_advectionProgress:
</pre><p>should it be %100?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ai2cm/pace/pull/246/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
I want it to print what step it's on every 100 steps. That way you get progress updates, but don't fill up the notebook with a ton of output.